packages/itwinui-react/src/core/Dialog/DialogContext.tsx

```diff
@@ -70,6 +70,10 @@ export type DialogContextProps = {
    * Dialog root ref. For internal use.
    */
   dialogRootRef?: React.RefObject<HTMLDivElement>;
+  /**
+   * Determines the positioning of Dialog on page.
+   */
+  position?: 'top-left' | 'top-right' | 'bottom-left' | 'bottom-right';
```

Comment:
should be called `placement`, not `position`

Reply:
done